src/installer/corehost/cli/hostmisc/pal.unix.cpp

```diff
@@ -118,7 +128,7 @@ void* pal::mmap_copy_on_write(const string_t& path, size_t* length)
 bool pal::getcwd(pal::string_t* recv)
 {
     recv->clear();
-    pal::char_t* buf = ::getcwd(nullptr, 0);
+    pal::char_t* buf = ::getcwd(nullptr, PAL_CWD_SIZE);
```

Comment:
Out of curiosity I've tested the getcwd(NULL, 0) on macOS and it works fine, despite the doc saying something else. But since it is an undocumented behavior, changing it for macOS too makes sense.